nshlib/nsh_proccmds.c

```diff
@@ -348,7 +348,7 @@ static int ps_callback(FAR struct nsh_vtbl_s *vtbl, FAR const char *dirpath,
   nsh_output(vtbl, "%3s ", status.td_cpu);
 #endif
 
-  nsh_output(vtbl, "%3s %-8s %-7s %3s %-8s %-9s ",
+  nsh_output(vtbl, "%3s %-8s %-7s %3s %-8s %-15s ",
```

Comment:
why need extend the width?

Reply:
do we still need this patch after we reduce the output length [https://github.com/apache/nuttx/pull/8556](https://github.com/apache/nuttx/pull/8556/files)

Reply:
don't need,hhh